dashboard/test/mailers/pd/workshop_mailer_test.rb

```diff
@@ -66,4 +66,320 @@ class WorkshopMailerTest < ActionMailer::TestCase
       Pd::WorkshopMailer.exit_survey(enrollment).deliver_now
     end
   end
+
+  test 'organizer should close workshop reminder email link is not relative path' do
+    workshop = create :pd_workshop, num_sessions: 1
+    mail = Pd::WorkshopMailer.organizer_should_close_reminder(workshop)
+
+    assert links_are_complete_urls?(mail)
+  end
+
+  test 'organizer cancel receipt email link is not relative path' do
+    workshop = create :pd_workshop, num_sessions: 1
```

Comment:
I see a lot of code reuse here. You might be able to make this a private method that takes a workshop and a method and does the links_are_complete assertion on the bottom. Not necessary so it's your call

Reply:
A lot of these can also probably be grouped into groups of test that pass in different courses